conans/client/command.py

```diff
@@ -831,6 +831,8 @@ def remove(self, *args):
                             help='Remove source folders')
         parser.add_argument("-l", "--locks", default=False, action="store_true",
                             help="Remove locks")
+        parser.add_argument("-sr", "--system-reqs", default=False, action="store_true",
```

Comment:
two points:
1. `-sr` has a confusion with combining `-s` (remove source folder) and `-r` (remote name). it probably should be `--sr` or `--system-reqs`. (I know, python command line parser might not support combining short arguments, but it's confusing for Unix users).
2. `Options are usually listed in alphabetical order unless this would make the utility description more confusing.` (http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html). so, I'd re-arrange them

Reply:
1. I would prefer remove then.
2. Agreed.

Reply:
Done!